ruby-challenges.rb

```diff
@@ -10,23 +10,94 @@
 filter_letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def certain_letter(array,letter)
+    new_arr = []
+
+    array.each do |word|
+        if word.include?(letter)
+            new_arr << word
+        end
+    end
+    p new_arr
+end
+
+# OUTPUTS
+# certain_letter(beverages_array, filter_letter_o)
+# Output: ["coffee", "soda water"]
+# certain_letter(beverages_array, filter_letter_t)
+# Output: ["tea", "water", "soda water"]
 
 # -------------------2) Create a method that takes in a hash and returns one array with all the hash values at their own index and in alphabetical order. No nested arrays. Use the test variable provided.
 # HINT: Google 'ruby get rid of nested arrays'
 
 us_states = { northwest: ['Washington', 'Oregon', 'Idaho'], southwest: ['California', 'Arizona', 'Nevada'], notheast: ['Maine', 'New Hampshire', 'Rhode Island'] }
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
+def alph_states(hash)
+    
+    p hash.values.flatten.sort
```

Comment:
I would remove the 'p' from any methods you make. These are the same as leaving console.logs in your Javascript functions